stub/src/stub_sai_host_interface.c

```diff
@@ -341,7 +341,7 @@ sai_status_t stub_host_interface_name_set(_In_ const sai_object_key_t      *key,
     return SAI_STATUS_SUCCESS;
 }
 
-sai_status_t stub_set_host_interface_trap_attribute(_In_ sai_hostif_trap_id_t  hostif_trapid,
+sai_status_t stub_set_host_interface_trap_attribute(_In_ sai_hostif_trap_type_t  hostif_trapid,
```

Comment:
there is a little bit confusion here. current, the sai thrift test is using the sonic branch header file instead of master.
since the master currently is keep changing until 1.0, it's probably not good to ask the test to track all the changes in the master.